fix: clarify requirements for setup steps in readme (#325)
MovedNetwork/op-move: README.md

# Integration testing

Make sure you have `go` installed on your system. Due to the pinned versions being based around August 2024, a version not older than 1.22 is required. Other dependencies include [foundry](http://getfoundry.sh/) for smart contract interaction and [jq](https://jqlang.github.io/jq/) being called indirectly by Optimism itself.

First, clone the Optimism monorepo. The repo is used to compile and deploy Optimism contracts.

```bash
git clone https://github.com/ethereum-optimism/optimism server/src/tests/optimism
```

> To pull in the libraries required by optimism, the following needs to be run from that repo:

Reviewer: We can use a direct tone:
```suggestion
To pull in the libraries required by Optimism, run the following inside that repo:
```
